iothub/service/src/ServiceClient.cs

```diff
@@ -42,11 +43,11 @@ internal ServiceClient()
         }
 
         /// <summary>
-        /// Create ServiceClient from the specified connection string
+        /// Create an instance of ServiceClient from the specified IoT Hub connection string.
         /// </summary>
-        /// <param name="connectionString">Connection string for the IoT Hub</param>
+        /// <param name="connectionString">Connection string for the IoT Hub.</param>
         /// <param name="options">The options that allow configuration of the service client instance during initialization.</param>
-        /// <returns></returns>
+        /// <returns>An instance of ServiceClient.</returns>
```

Comment:
use <see cerf="ServiceClient">

Reply:
I remember receiving feedback in previous PRs that linking the same type in a doc comment for the type was redundant; i.e. linking ServiceClient in doc comments for ServiceClient.
Let me know if you feel otherwise.

Reply:
I see we follow this pattern everywhere, so pointed it out. This could be a good topic for parking lot. We do the same in track 2 repos as well so would be good be consistent.